packages/flutter/test/material/date_picker_test.dart

```diff
@@ -449,14 +449,29 @@ void _tests() {
               TestSemantics(
                 children: <TestSemantics>[
                   TestSemantics(
+                    id: 55,
                     actions: <SemanticsAction>[SemanticsAction.scrollLeft, SemanticsAction.scrollRight],
                     children: <TestSemantics>[
                       TestSemantics(
                         children: <TestSemantics>[
                           TestSemantics(
                             children: <TestSemantics>[
                               TestSemantics(
+                                id: 11,
+                                flags: <SemanticsFlag>[SemanticsFlag.hasImplicitScrolling],
                                 children: <TestSemantics>[
+                                  TestSemantics(),
```

Comment:
Why do we have all these extra empty nodes? They really shouldn't be here...

Reply:
I believe they're containers/placeholders to make the calendar day fall under the right slot.

Reply:
Empty placeholders should not have a semantics node, though. I thought we filter those out...

Reply:
They're from the `ExcludeSemantics` around the weekday header and more empty nodes for the containers for placeholders.

So 7 weekday headers + 5 placeholder containers = 12 empty semantics nodes.

Reply:
I'm fine with changing that but I think if we do it should be in a separate PR.

Reply:
OK, Can you add a TODO and file an issue so it doesn't get lost?

Reply:
https://github.com/flutter/flutter/issues/34431